_source/logzio_collections/_prometheus-sources/telegraf-nvidia.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ship NVIDIA SMI metrics via Telegraf
+logo:
+  logofile: nvidia-telegraf.png
+  orientation: vertical
+data-source: NVIDIA SMI metrics over Telegraf
+templates: ["docker"]
+contributors:
+  - daniel-tk
+  - nshishkin
+shipping-tags:  
+  - prometheus
+order: 800
+---
+
+
+## Overview
+
+Telegraf is a plug-in driven server agent for collecting and sending metrics and events from databases, systems and IoT sensors.
+
+To send your Prometheus-format NVIDIA SMI metrics to Logz.io, you need to add the **inputs.nvidia_smi** and **outputs.http** plug-ins to your Telegraf configuration file.
+
+#### Configuring Telegraf to send your metrics data to Logz.io
+
+<div class="tasklist">
+
+##### Set up Telegraf v1.17 or higher
```

Comment:
only for linux and need to tell customer to install it on the same machine as the NVIDIA card